src/main/scala/strawman/collection/immutable/TreeSet.scala

```diff
@@ -113,4 +115,5 @@ object TreeSet extends SortedIterableFactory[TreeSet] {
       case _ => empty[E] ++ it
     }
 
+  def newBuilder[A : Ordering](): Builder[A, TreeSet[A]] = new ArrayBuffer[A].mapResult(sortedFromIterable[A] _)
```

Comment:
I'll address those issues in the Set PR. Doing it the old way requires a SetBuilder, which in turn requires a `+` operation on Sets that we do not have yet...